extensions/cornerstone/src/commandsModule.ts

```diff
@@ -267,6 +267,18 @@ function commandsModule({
       toolbarService.recordInteraction(props);
     },
 
+    // Enable or disable a toggleable command, without calling the activation
+    // Used to setup already active tools from hanging protocols
+    toolbarSetActive: props => {
+      toolbarService.setActive(props.toolId, props.isActive ?? true);
+    },
```

Comment:
I think we should rename this function in the toolbarService. It is not setActive it is basically `updateToggleStatus`

```
/** Sets the toggle state of a button to the isActive state */
  public setActive(id: string, isActive: boolean): void {
    if (isActive) {
      this.state.toggles[id] = true;
    } else {
      delete this.state.toggles[id];
    }
  }
```

Reply:
Renamed to setToggled to be in keeping with the existing naming.

Reply:
I don't like setToggled and don't like updateToggleStatus either, @jbocce what do you think

Reply:
I prefer to leave it as `setActive` because it is most explicit as such.

Reply:
But it is not actually setActive, it acts on only toggles, so toggles should be on the name at least

Reply:
In my opinion it IS `setActive`. Consider using `setActiveForToggleButton`.